Chrysalis-Papilio/Soen-490-Papilio-Android
# Users should not be able to join and favorite their own activities

# General info

**Issue number**: PAPILIO-109

**Task description**: 
 - See title

# Testing
To test this feature you have to:

1. Create an activity with your user account
2. Click on that same activity in the main feed
3. On the details page, the join and favorite buttons should not be displayed

# Notes
- Testing will follow soon in a separate PR
